### DISCUSSION TITLE
CommunityToolkit support for Trimming added with .NET 5

### DISCUSSION THREAD

**Discussion by paulober** [CommunityToolkit/dotnet#556] [Q&A] [answered]
Does CommunityToolkit for WinuI3 now support `<PublishTrimmed>True</PublishTrimmed>`  .NET feature?

**Comment by Sergio0694** [comment, marked as answer]
The .NET Community Toolkit has been fully trimmable and with trimming annotations since 8.0, long before WinUI supported trimming 🙂
